src/store/reducers/isAuthorized.js

```diff
@@ -12,8 +12,6 @@ export default function isAuthorized(state = initialState, action) {
       return { ...state, isAuthorized: true };
     case USER_LOGIN.FAILURE:
       return { ...state, error: action.payload };
-    case USER_LOGOUT.SUCCESS:
```

Comment:
Are you sure?

Reply:
When we use "go" after clearing the localhost, the value isAuthorized is automatically set to default without calling an action